src/pages/home/HeaderView.js

```diff
@@ -155,7 +155,7 @@ function HeaderView(props) {
     );
 
     const canJoinOrLeave = isChatThread || isUserCreatedPolicyRoom || canLeaveRoom;
-    const canJoin = canJoinOrLeave && !isWhisperAction && props.report.notificationPreference === CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN;
+    const canJoin = canJoinOrLeave && !isWhisperAction && props.report.notificationPreference === CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN && props.report && props.report.reportID;
```

Comment:
Revert this.

Instead use the `isLoading` value in the below rendering condition 

```js
            {canJoin && isSmallScreenWidth && <View style={[styles.ph5, styles.pb2]}>{joinButton}</View>}
```